content/chapters/compute/lab/content/processes.md

```diff
@@ -114,17 +114,17 @@ From the outside, it's as if you were running these commands from the terminal.
 Head over to `support/sleepy/sleepy_creator.py`.
 Use `subprocess.Popen()` to spawn 10 `sleep 1000` processes.
 
-1. Now use the same `pstree -pac` command and look for `sleepy_creator.py`.
+1. Now use `ps -e -H -o pid,ppid,cmd | (head -1; grep "python3 sleepy_creator.py")` command and look for `sleepy_creator.py`.
```

Comment:
First instruct to start the script using `python sleepy_creator.py`.

Place all commands under a code block (triple backticks). use `console` text type for code blocks with shell commands.